src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -687,6 +687,12 @@ void program::transfer_memory_to_device() {
             }
 
             if (alloc_type == allocation_type::usm_host || alloc_type == allocation_type::usm_shared) {
+                // usm_device memory does not provide performance benefits on the LNL platform
+                if (get_engine().get_device_info().arch == gpu_arch::xe2 &&
+                    get_engine().get_device_info().dev_type == device_type::integrated_gpu) {
```

Comment:
do we apply this change only for constant? why not for all other allocated activations memories?